synapse/storage/databases/main/stream.py

```diff
@@ -1401,7 +1401,7 @@ def _paginate_room_events_txn(
             `to_token`), or `limit` is zero.
         """
 
-        args = [False, room_id]
+        args = [room_id]
```

Comment:
```suggestion
        args: List[Any] = [room_id]
```


Failure in CI: https://github.com/matrix-org/synapse/actions/runs/5488412327/jobs/10002270648?pr=15525#step:7:15
```
synapse/storage/databases/main/stream.py:1425: error: Argument 1 to "append" of "list" has incompatible type "int"; expected "str"  [arg-type]
```